ghost/core/test/unit/frontend/public/ghost-stats.test.js

```
@@ -1,688 +1,323 @@
+        const {GhostStats} = require(path.resolve(__dirname, '../../../../core/frontend/src/ghost-stats/ghost-stats.js'));
+        ghostStats = new GhostStats(browserService);
     });
 
-    describe('Tinybird object creation', function () {
-        it('should create a Tinybird object', function () {
-            expect(env.window.Tinybird).to.exist;
-            expect(env.window.Tinybird.trackEvent).to.be.a('function');
-        });
+    afterEach(function () {
+        sandbox.restore();
+        delete global.window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)